sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -87,6 +87,27 @@ internal static LusidInstrument CreateExampleBond()
                 instrumentType: LusidInstrument.InstrumentTypeEnum.Bond
             );
 
+        internal static LusidInstrument CreateExampleZeroCouponBond()
+            => new Bond(
+                startDate: new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero),
+                maturityDate: new DateTimeOffset(2020, 9, 18, 0, 0, 0, TimeSpan.Zero),
+                domCcy: "USD",
+                principal: 100m,
+                couponRate: 0m,
+                flowConventions: 
+                    new FlowConventions(
+                        currency: "USD",
+                        paymentFrequency: "0Invalid",
```

Comment:
what does this mean? where does a user get a list of fields that they can use? we need to add either comments for them to resolve or links to a KB article

Reply:
Unfortunetelly right now this is the only way how a ZCB schedule can be created. `0Invalid` is 

- not documented
- will be replaced in future